.github/workflows/conformance-ipsec-e2e.yaml

```diff
@@ -304,9 +304,9 @@ jobs:
         uses: ./.github/actions/ipsec-key-rotate
         with:
           key-algo: ${{ matrix.key-two }}
-          key-type-one: ${{ matrix.key-type-one }}
-          key-type-two: ${{ matrix.key-type-two }}
           encryption-overlay: ${{ matrix.encryption-overlay }}
+          nb-nodes: 2
+          ipv6: true
```

Comment:
It took me a while to understand that a parameter not in matrix is as equal as if it was set to true and that this is actually the expected behavior for this test :+1: 

While [setting up Kind](https://github.com/cilium/cilium/pull/35806/files#diff-b77d9c4420c99e9bbac5afc52278677e846739ed4b7b4f7c1efdb24f1b69a249L189) we do [use](https://github.com/cilium/cilium/blob/6a9766b37af22f6b902e4acbf8a9f137f545b35b/.github/workflows/conformance-ipsec-e2e.yaml#L172
) `$matrix.ipv6`, but we don't have it set in [.github/actions/ipsec/configs.yaml](https://github.com/cilium/cilium/blob/6a9766b37af22f6b902e4acbf8a9f137f545b35b/.github/actions/ipsec/configs.yaml), hence resulting in `ipv6=""`.

[Later](https://github.com/cilium/cilium/actions/runs/11704540713/job/32597494835#step:9:2) in In `/ci-ipsec-e2e`:
```bash
IP_FAM="dual"
if [ "" == "false" ]; then
  IP_FAM="ipv4"
fi
echo params="--xdp --secondary-network \"\" [3](https://github.com/cilium/cilium/actions/runs/11704540713/job/32597494835#step:9:3) \"\" \"\" iptables $IP_FAM" >> $GITHUB_OUTPUT
```
resulting in ipv6 being used and that is why `ipv6: true`, right?

Reply:
Yep, that's correct. The "IPv6 enabled by default" also puzzled me a bit, but I guess it matches what the agent does as well.